inst/include/Rcpp.h

```diff
@@ -80,4 +80,6 @@
 
 #include <Rcpp/platform/solaris.h>
 #include <Rcpp/api/meat/meat.h>
+
+#include <Rcpp/algorithms.h>
```

Comment:
nit: a non-plural name is slightly preferred (e.g. `<Rcpp/algorithm.hpp>`); just to be in-line with the STL `<algorithm>` header. Also since the namespace name is `algorithm` after all.